docs/examples/fd_fitting_basic/index.rst

```diff
@@ -0,0 +1,194 @@
+
+And fit the model::
+
+    >>> fit_odijk.fit()
+
+    Fit
+      - Model: DNA(x-d)_with_DNA
+      - Equation:
+          f(d) = argmin[f](norm(DNA.Lc * (1 - (1/2)*sqrt(kT/(f*DNA.Lp)) + f/DNA.St)-(d - DNA.d_offset))) + DNA.f_offset
+
+      - Data sets:
+        - FitData(Inverted Odijk, N=959)
+
+      - Fitted parameters:
+        Name                  Value  Unit      Fitted      Lower bound    Upper bound
+        ------------  -------------  --------  --------  -------------  -------------
+        DNA/d_offset    -0.00601252  [au]      True              -0.01           0.01
+        DNA/Lp          44.2558      [nm]      True               0            100
+        DNA/Lc           2.80085     [micron]  True               0            inf
+        DNA/St        1714.56        [pN]      True               0            inf
+        kT               4.11        [pN*nm]   False              0              8
+        DNA/f_offset     0.0392458   [pN]      True              -0.1            0.1
+
+Set up the Twistable worm like chain model
+------------------------------------------
+
+Set up a twistable worm like chain model with a distance and force offset. By default,
+the `twistable_wlc` model provided with pylake is defined as distance as a function of
+force. Typically, we want to fit force as a function of distance however. To achieve
+this, we can invert the model using its `invert` function. However, this can be slow
+in certain cases, as it requires an inversion for each data point. Luckily we have a
+faster way of achieving this in pylake, which is to use the dedicated `inverted_twistable_wlc`
+model.
+
+Again, we incorporate an offset in both distance and force to compensate for small
+offsets that may exist in the data:::
+
+    m_dna = pylake.inverted_twistable_wlc('DNA').subtract_independent_offset() + pylake.force_offset('DNA')
+    fit_twlc = pylake.FdFit(m_dna)
+
+Load the data into the model
+----------------------------
+
+Now it is time to load the full data into the model. Note in the figure however,
+that there seems to be a small break at the end of the Fd curve. The model will
+not be able to capture this behaviour, and therefore it is best to remove it
+prior to fitting as it will otherwise bias our parameter estimates::
+
+    force = fd_curve.f.data
+    distance = fd_curve.d.data
+    mask = distance < 2.88
+    distance = distance[mask]
+    force = force[mask]
+
+Now we can load the data into the model::
+
+    fit_twlc.add_data("Twistable WLC", force, distance)
+
+We could add more datasets in a similar manner, but in this example, we only fit a single model. Let’s load the
+parameters from our previous fit to use them as initial guesses for this one. We also fix the twist rigidity and
+critical force values to values from literature (analogous to Broekmans et al. "DNA twist stability changes with
+magnesium (2+) concentration." Physical review letters 116.25 (2016))::
```

Comment:
😁

> Docs file structure comments:
> 
> * Try to name the `.rst` files so that they are related to the page title. Right now you have `Twistable Worm-Like-Chain Fitting` in a file called `fd_fitting_basic`. Something like `twlc_fitting` might be a better name. These names end up in the URL so it can be confusing if there is a mismatch.

Renamed. Actually, this was a bit of a pet peeve to begin with, because one annoying thing was that if you downloaded the file as jupyter notebook, it would be named index.rst which is a annoying. I renamed the other examples too.

> * `fd_fitting_advanced` seems to be in the file structure but it's not in the `toctree` so it's not rendered in the HTML. Is this intentional?

I removed it for now. While that notebook would be great to have in the future I think, it depends on some yet unreleased piezo tracking code. I have a reasonably clean implementation of that, but we should discuss on how to review that, and whether we also want that in Pylake. Until that time, I will just leave the notebook out.